pkg/util/core_windows.go

```diff
@@ -0,0 +1,11 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package util
+
+// SetCoreLimit enables core dumps and sets the core dump size limit based on configuration
+func SetCoreLimit() error {
+	return nil
```

Comment:
this could return an error such as `"unsupported on Windows"`

Reply:
✅